react/ui-components/src/hoc/EditablePopup.js

```
@@ -0,0 +1,130 @@
+      onOverlayClick={() => {
+        setShowEditablePopup(false);
+        setEditRow(null);
+      }}
+      children={[
+        <FormComposer
+          label={t('Submit')}
+          config={transformConfig(config ? config : {})}
+          defaultValues={generateDefaultValues(config,editRow)}
+          onSubmit={onSubmit}
+          labelfielddirectionvertical={false}
+        />,
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)